docs/api/ctx.md

```diff
@@ -1709,7 +1709,7 @@ app.Get("/", func(c fiber.Ctx) error {
 
 ## SendFile
 
-Transfers the file from the given path. Sets the [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) response HTTP header field based on the **filenames** extension.
+Transfers the file from the given path. Sets the [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) response HTTP header field based on the **file format**.
```

Comment:
I believe this should say "file extension".

Reply:
fixed in 6df30b5df8a1e0fbcfd51f79b15746d9d001ed12